src/components/pages/error-page.tsx

```diff
@@ -89,12 +90,11 @@ export function ErrorPage({ code, message }: ErrorPageProps) {
   function renderHomeLink() {
     //no csr here
     return (
-      <a
-        href="/"
-        className="serlo-button serlo-make-interactive-primary mr-4 mt-4"
-      >
-        {strings.errors.backToHome}
-      </a>
+      <Link href="/">
```

Comment:
again this was intentional to include a save refresh after erroring.

Reply:
ah, this looks like another use-case, in which case a prop like `forceNoCSR` is indead useful.
-> and a prop like `unstyled` to avoid using the `UnstyledLink` component